packages/homepage/content/changelog/3-6-2/index.md

```diff
@@ -0,0 +1,50 @@
+---
+banner:
+title: v3.6.2
```

Comment:
Shouldn't this be `3.7.0`, since it adds features? 🤔

Reply:
As marked these are not considered featured